shop/src/main/java/ro/msg/learning/shop/model/Revenue.java

```diff
@@ -0,0 +1,45 @@
+package ro.msg.learning.shop.model;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+import lombok.experimental.SuperBuilder;
+import ro.msg.learning.shop.dto.RevenueForLocation;
+
+import javax.persistence.*;
+import java.math.BigDecimal;
+import java.time.LocalDate;
+
+@NamedNativeQuery(name = "Revenue.findAllRevenuesPerLocation",
```

Comment:
The right place for queries is in the Repository. Please move this query there and keep the model entity clean and readable.